queries/djot/highlights.scm

```diff
@@ -85,106 +90,168 @@
 ] @markup.list
 
 (list_marker_task
-  (unchecked) @constant.builtin) @markup.list.unchecked
+  (unchecked)) @markup.list.unchecked
 
 (list_marker_task
-  (checked) @constant.builtin) @markup.list.checked
+  (checked)) @markup.list.checked
+
+; Colorize `x` in `[x]`
+((checked) @constant.builtin
+  (#offset! @constant.builtin 0 1 0 -1))
 
 [
   (ellipsis)
   (en_dash)
   (em_dash)
-  (straight_quote)
 ] @string.special
 
+(list_item
+  (term) @type.definition)
+
+(quotation_marks) @string.special
+
+; Conceal { and } but leave " and '
+((quotation_marks) @string.special
+  (#eq? @string.special "{\"")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "\"}")
+  (#offset! @string.special 0 1 0 0)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "{'")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#eq? @string.special "'}")
+  (#offset! @string.special 0 1 0 0)
+  (#set! conceal ""))
+
+((quotation_marks) @string.special
+  (#any-of? @string.special "\\\"" "\\'")
+  (#offset! @string.special 0 0 0 -1)
+  (#set! conceal ""))
```

Comment:
These can be combined into 2 patterns

```suggestion
; Conceal { and } but leave " and '
((quotation_marks) @string.special
  (#any-of? @string.special "{\"" "{'" "\\\"" "\\'")
  (#offset! @string.special 0 0 0 -1)
  (#set! conceal ""))

((quotation_marks) @string.special
  (#any-of? @string.special "\"}" "'}")
  (#offset! @string.special 0 1 0 0)
  (#set! conceal ""))
```

Also, are these like markdown backticks, or string quotes? If the latter, then I don't think we should hide it

Reply:
These are string to control start/end quotes like `“` and `”` (`{'` forces the "left" variant).

Reply:
Yeah, don't conceal quotes. (Markdown is special and gets a pass.) If in doubt, leave it for users to add.

Reply:
Ah, djot is also a markdown-style language; that changes things. Then you can use `conceal` to "render" it as much as you want.

Reply:
Can you group some of these queries @treeman. I don't think it's necessary to separate these captures into 6 queries, 2 queries + `any-of` should be enough

Reply:
Yes sorry about that, I thought i accepted the suggestion but I must have screwed up somehow.

Fixed not.